src/shared/varlink-io.systemd.Machine.c

```diff
@@ -34,7 +34,8 @@ static SD_VARLINK_DEFINE_METHOD(
                 SD_VARLINK_DEFINE_INPUT(id,                SD_VARLINK_STRING, SD_VARLINK_NULLABLE),
                 SD_VARLINK_DEFINE_INPUT(service,           SD_VARLINK_STRING, SD_VARLINK_NULLABLE),
                 SD_VARLINK_DEFINE_INPUT(class,             SD_VARLINK_STRING, 0),
-                SD_VARLINK_DEFINE_INPUT_BY_TYPE(leader,    ProcessId,         SD_VARLINK_NULLABLE),
+                SD_VARLINK_DEFINE_INPUT(leader,            SD_VARLINK_INT,    SD_VARLINK_NULLABLE),
+                SD_VARLINK_DEFINE_INPUT_BY_TYPE(leaderPid, ProcessId,     SD_VARLINK_NULLABLE),
```

Comment:
alignment is off

Reply:
hmm, would be good if we add comments for at least these two fields now.
```c
SD_VARLINK_FIELD_COMMENT("The leader PID as simple positive integer.");
```

and 

```
SD_VARLINK_FIELD_COMMENT("The leader PID as ProcessId structure. If both the leader and leaderPid parameters are specified they must reference the same process. Typically one would only specify one or the other however. It's generally recommended to specified leaderPid as it references a process in a robust way without risk of identifier recycling.");
```

or something like that?

Reply:
maybe name the 2nd field leaderProcessid, to match the type name?